how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -0,0 +1,126 @@
+
+## Pre-requisites
+
+This document requires ACRN hypervisor and Kata containers installed on your system. The following provide links to installation of ACRN hypervisor and Kata containers:
+
+- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
+- ACRN [software](https://projectacrn.github.io/latest/getting-started/apl-nuc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- Kata containers installation: Automated installation doesn't seem to be supported for clearlinux, so please use [manual installation](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md) steps.
+
+> **Note:** Create rootfs image and not initrd image.
+
+In order to run Kata + ACRN, your container stack must provide block-based storage ('graph driver'), such as device-mapper.
+
+> **Note:** Currently only 1 VM can be launched from kata-containers using ACRN hypervisor (SDC scenario) due to [this issue](https://github.com/kata-containers/runtime/issues/1785). Once the issue is addressed from kata, we can test with other post launched VM. Other scenarios (such as Industry, hybrid, and logical partitioning) are not supported yet.
+
+## Install and Configure Docker
+
+Docker 18.06 is required for running Kata with ACRN. For Kata+ACRN, a block-based driver like device-mapper is required. The latest release of Docker, 18.09, does not support device-mapper and is not compatible.
+
+To configure Docker for device-mapper and Kata, first, stop Docker daemon if it is already running using,
+
+```bash
+sudo systemctl stop docker
+```
+
+Set /etc/docker/daemon.json with the following contents:
+
+```bash
+{
+  "storage-driver": "devicemapper"
+}
+```
+
+Restart docker:
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+```
+
+Configure [Docker](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#update-the-docker-systemd-unit-file) to use kata-runtime
+
+## Configure Kata Containers with ACRN
+
+To configure Kata Containers with ACRN, all that is needed is to copy the configuration-acrn.toml file (please find attached) that is generated when building the kata-runtime to either /etc/kata-containers/configuration.toml or /usr/share/defaults/kata-containers/configuration.toml.
```

Comment:
Find attached where?

Reply:
I have removed this statement. Good catch!